public/css/style.css

```diff
@@ -32,7 +32,7 @@ body a:hover{
 }
 /*--view--*/
 .view{
-    margin-top:3.5%;
+    margin-top: 50px;
```

Comment:
this won't be applicable to all the screen sizes. Did you test on all the screen sizes/ resolutions ?